grails-app/services/icaruswings/CustomerService.groovy

```diff
@@ -0,0 +1,18 @@
+package icaruswings
+
+import grails.gorm.transactions.Transactional
+import icaruswings.utils.PersonType
+
+@Transactional
+class CustomerService {
+    public Customer save(String name, String email, String cpfCnpj) {
+        Customer customer = new Customer()
+        customer.name = name
+        customer.email = email
+        customer.cpfCnpj = cpfCnpj
+        customer.personType = PersonType.NATURAL
```

Comment:
Por que fixar? Mesma pergunta vale pro Payer

Reply:
Por enquanto a validação de PersonType com cpf e cnpj ainda não tinha sido criada

Reply:
Então podes criar apenas quando o tratamento for implementado